domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt

```diff
@@ -0,0 +1,8 @@
+﻿{
+      "Bearer": {
+        "type": "http",
+        "description": "JWT Authorization header using the Bearer scheme. \r\n\r\n Enter 'Bearer' [space] and then your token in the text input below.\r\n\r\nExample: \"Bearer 1safsfsdfdfd\"",
```

Comment:
Are these \r n and needed?

Reply:
I will check :)

Reply:
Without /r + /n:

![image](https://github.com/Energinet-DataHub/energy-origin/assets/55732914/c25be62d-b578-420b-a497-42f61dc63fd4)


With:

![image](https://github.com/Energinet-DataHub/energy-origin/assets/55732914/3427a6ab-c034-418b-ae32-33480d711b14)

You decide.